_posts/2018-10-18-TDD-is-not-about-tests.md

```diff
@@ -13,11 +13,11 @@ Contrary what the name might make most people think, **test-driven development i
 
 By tackling work in smaller increments we reduce the size of the problems we have to solve and thus reduces the scope of the decisions we need to take. Decreasing the size and quantity of problems then helps us think more about what is the best solution for each one of them.
 
-We can apply this methodology to many other areas of software engineering and, in fact, it is. If you take a look at what we do with stories on agile methodologies by breaking them down into multiple tasks you will see that in fact, we're just reducing the size and the scope of the problems we're trying to solve so that we can solve them better.
+We can apply this methodology to many other areas of software engineering and, in fact, it already has been. If you take a look at what we do with stories on agile methodologies by breaking them down into multiple tasks you will see that in fact, we're just reducing the size and the scope of the problems we're trying to solve so that we can solve them better.
 
 **Test-driven development is about making writing code an iterative process**.
 
-If you want to build a car you should not start by assembling the wheels, then adding the chassis, the engine and the interior. Before building a car you need to know how to build a skate.
+If you want to build a car you should not start by assembling the wheels, before adding the chassis, the engine and the interior. Before building a car you need to know how to build a skateboard.
```

Comment:
I think that this `before` here is not exactly what I meant. I meant that this is the wrong order, not that assembling the wheels before the rest is wrong.